tests/run_container.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

given the rest of the script, this has to be bash not sh - at least `$()` won't work in plain sh, and I'm not sure about all those `${:+}` either.

Reply:
I suspect the original author wrote this with `sh` as a symlink to `bash`. Have updated it point directly to bash.